### PR TITLE
expose reventlog and modify time format

### PR DESCRIPTION
**Pull request by xuweibj**
#3497 
1. Expose reventlog in openbmc.pm

2. Change ``Timestamp`` format to UTC time.
```
# XCATBYPASS=1 reventlog openbmc156 5 -s
[OpenBMC development support] Using this version of xCAT, ensure firware level is at v1.99.6-0-r1, or higher.
openbmc156: Entry 67: 07/26/2017 19:13:59 org.open_power.Host.Error.WatchdogTimedOut
openbmc156: Entry 66: 07/26/2017 19:13:48 org.open_power.Host.Error.Checkstop
openbmc156: Entry 65: 07/26/2017 19:12:49 org.open_power.Host.Error.Checkstop
openbmc156: Entry 64: 07/26/2017 19:11:51 org.open_power.Host.Error.Checkstop
openbmc156: Entry 63: 07/26/2017 19:10:54 org.open_power.Host.Error.Checkstop
```

* [ ]  For option ``clear``, it is to delete all logs. There is no interface now, it will be done in phase 5 depending on comments in https://github.com/openbmc/openbmc/issues/2001.

### CODE REVIEW

**Comment by zet809** [comment]
Looks good to me, I agree to merge.

**Comment by whowutwut** [comment]
👍 Thanks @xuweibj !

File: xCAT-server/lib/xcat/plugins/openbmc.pm [hunk 1]
```diff
@@ -135,8 +135,8 @@ my %status_info = (
     },
     REVENTLOG_CLEAR_REQUEST => {
         method         => "POST",
-        init_url       => "$openbmc_url/records/events/action/clear",
-        data           => "",
+        init_url       => "$openbmc_project_url/logging//action/delete",
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/openbmc.pm]: Is that extra / intentional?

File: xCAT-server/lib/xcat/plugins/openbmc.pm [hunk 2]
```diff
@@ -135,8 +135,8 @@ my %status_info = (
     },
     REVENTLOG_CLEAR_REQUEST => {
         method         => "POST",
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/openbmc.pm]: We should be using the DELETE call for deleting... not POST.

**Comment by whowutwut** [comment]
In order to fully expose, we need to get rid of these lines: 
```
[root@stratton01 frame6u05]# diff /opt/xcat/lib/perl/xCAT_plugin/openbmc.pm /opt/xcat/lib/perl/xCAT_plugin/openbmc.pm.orig
766a767,768
>             xCAT::SvrUtils::sendmsg("Command $command is not available now!", $callback);
>             return 1;
```

**Pull request merged**